app/model/sdl/VehicleInfoModel.js

```diff
@@ -210,7 +210,11 @@ SDL.SDLVehicleInfoModel = Em.Object.create(
       'fuelRange': [
         {
           'type':'GASOLINE',
-          'range': 400
+          'range': 400,
+          'level': 10,
+          'levelState': 'NORMAL',
+          'capacity': 100,
+          'capacityUnit': 'KILOGRAMS'
```

Comment:
```suggestion
          'capacityUnit': 'LITERS'
```
Please refer to the `capacityUnit` param description in the `FuelRange` struct
> The unit of the capacity of this fuel type such as liters for gasoline or kWh for batteries.